docs/performance.md

```diff
@@ -58,10 +58,10 @@ service
 | :---:             | :---:           | :---:       | :---:             | :---:                  |
 | OpenTelemetry        | 1               | 2           | ~9K               | 8K                     |
 | OpenTelemetry        | 2               | 4           | ~18K              | 16K                    |
-| Jaeger Thrift     | 1               | 6           | ~11.5K            | 10K                    |
-| Jaeger Thrift     | 2               | 8           | ~23K              | 20K                    |
-| Zipkin v2 JSON    | 1               | 6           | ~8.5K             | 7K                     |
-| Zipkin v2 JSON    | 2               | 8           | ~16K              | 14K                    |
+| Jaeger Thrift     | 1               | 2           | ~11.5K            | 10K                    |
```

Comment:
Looking at the history of this line, I would guess the OpenTelemetry one is the one that should be changed: https://github.com/open-telemetry/opentelemetry-collector/commit/1ce867d2716d40b24e614ea87be183c087a9ca30

I wouldn't change this without you having executed new performance tests or confirming with the original author if the numbers are wrong.

Reply:
@jpkrohling You are right. I apologize, it this was a little hasty of me not to check previous history and assume there was a typo. 

I will draft this PR for now and check with author and previous contributors. If currently everything is correct, I will update the docs with an explanation.